lib/error_lib/error.go

```diff
@@ -0,0 +1,28 @@
+package error_lib
+
+import (
+	"fmt"
+	"todo-clean/common"
+	"todo-clean/delivery/model"
+)
+
+func WrapError(input string, inputErr error) (err error) {
+	err = fmt.Errorf("%s error: %w", input, inputErr)
+	return err
+}
+
+func CheckEmptyStringCreateTodoRequest(input model.CreateTodoDeliveryRequest) (err error) {
```

Comment:
Note: you can use package "github.com/go-playground/validator/v10" for input validation